docs/fundamentals/runtime-libraries/snippets/System/Single/Overview/csharp/precisionlist4a.cs

```diff
@@ -1,13 +1,13 @@
-﻿// <Snippet17>
-using System;
+﻿using System;
 using System.IO;
 
 public class Example10
 {
     public static void Main()
     {
-        StreamWriter sw = new StreamWriter(@".\Singles.dat");
-        Single[] values = { 3.2f / 1.11f, 1.0f / 3f, (float)Math.PI };
+        // <Snippet17>
+        StreamWriter sw = new(@".\Singles.dat");
```

Comment:
This path won't work on linux / mac:

```suggestion
        StreamWriter sw = new(@"./Singles.dat");
```